corecomponent/androidcomponent/src/main/java/io/github/droidkaigi/confsched2020/ui/ChromeCustomTabsNavigator.kt

```diff
@@ -39,7 +39,7 @@ class ChromeCustomTabsNavigator(private val context: Context) :
         val builder = CustomTabsIntent.Builder()
             .setShowTitle(true)
             .enableUrlBarHiding()
-            .setToolbarColor(context.getThemeColor(R.attr.colorAccent))
+            .setToolbarColor(ContextCompat.getColor(context, R.color.custom_tabs_toolbar))
```

Comment:
Thank you!
How about using colorSurface?

```
        <item name="colorSurface">@color/white</item>
```

night

```
        <item name="colorSurface">@color/black_121212</item>
```

Reply:
You're right.
I would update again using colorSurface.